### ISSUE TITLE
How to include another CSL?

### ISSUE THREAD

**Issue by adebali** [manubot/rootstock#242]
I wasn't able to find in the documents how I can implement another CSL file to render my references based on some specific requirements.

I think modifying the line `CSL_PATH=build/assets/style.csl` in `build.sh` is the way. Is there an easier way without changing `build.sh`?

**Comment by dhimmel** [comment]
Our current recommendation is to edit the line in `build.sh`:

https://github.com/manubot/rootstock/blob/ed4158c9f2a1b300f82d058be9aefe1f5eb72763/build/build.sh#L23

You should be able to point this at a URL. You can find available styles at https://github.com/citation-style-language/styles or https://www.zotero.org/styles. If using a style from GitHub, make sure to replace `blob` with `raw` in the URL to get the raw file as opposed to the HTML page.

You can also edit `build/assets/style.csl`, but I'd only recommend this for small edits. You could also make a new local file like `build/assets/custom-style.csl` and then change `build.sh` to point at this.

**Comment by agitter** [comment]
Let's add these instructions to `USAGE.md`.

We may also want to revisit #189 to discuss longer term plans for customizing the build process (CSL, plugins, etc.) through a config file, environment variables, or something else that is less technical than editing a bash script.

**Comment by ben-heil** [comment]
For any future Manubot users who came here because this is the top Google result for "Manubot CSL", this fix is no longer valid. The updated method for changing CSL files can be found here: https://github.com/manubot/rootstock/issues/334#issuecomment-620053989

tl;dr go to `build/pandoc/defaults/common.yaml` and update the `csl` field there

**Comment by agitter** [comment]
Thanks @ben-heil.

To help keep this issue future proof in case the behavior changes again, we also describe the way to do this in the "Custom formatting" section of `USAGE.md`, which is available [here](https://github.com/manubot/rootstock/blob/main/USAGE.md#custom-formatting).

**Comment by lubianat** [comment]
@agitter as a note here, the manifesto "USAGE.md" is outdated, and still points to build.sh: 
https://github.com/manubot/manufesto/blob/master/USAGE.md

**Comment by dhimmel** [comment]
Ah I can also update manufesto. I expect the underlying issue here is that @lubianat googled something like "manubot usage github" and the rootstock USAGE is not even listed on the first page! I wonder why.
